### DISCUSSION TITLE
What should I do?

### DISCUSSION THREAD

**Discussion by Sunrongguo2008** [LSPosed/MagiskOnWSALocal#139] [Q&A] [answered]
![Screenshot_20220924_173135](https://user-images.githubusercontent.com/91547012/192090997-c95ee329-c726-4fa1-9d82-9fff91120717.jpg)
Ubuntu 22.04 LTS stable

**Comment by Howard20181** [comment, marked as answer]
你的软件源有问题，不要用tuna的
试试腾讯或阿里的
